clickhouse/include/userver/storages/clickhouse/io/impl/escape.hpp

```diff
@@ -22,26 +23,35 @@ std::string Escape(int16_t);
 std::string Escape(int32_t);
 std::string Escape(int64_t);
 
+std::string Escape(const char* source);
+std::string Escape(const std::string& source);
 std::string Escape(std::string_view source);
 
 std::string Escape(std::chrono::system_clock::time_point source);
 std::string Escape(DateTime64Milli source);
 std::string Escape(DateTime64Micro source);
 std::string Escape(DateTime64Nano source);
 
-template <typename T>
-std::string Escape(const std::vector<T>& source) {
-  // just an approximation, wild guess
-  constexpr size_t kResultReserveMultiplier = 5;
+template <typename Container>
+std::string Escape(const Container& source) {
+  static_assert(traits::kIsRange<Container>,
+                "There's no escape implementation for the type.");
```

Comment:
for non-range types

and please add the type name

Reply:
> and please add the type name

probably we don't need it as Container is already included in the error message

Reply:
> > and please add the type name
> 
> probably we don't need it as Container is already included in the error message

Yeap, it is included